.github/workflows/code-checkers.yml

```diff
@@ -2,12 +2,16 @@ name: Static code checkers
 
 on: [push]
 
+permissions: {}
+
 jobs:
   mypy:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: astral-sh/setup-uv@v6
+      with:
+        persist-credentials: false
+    - uses: astral-sh/setup-uv@bd01e18f51369d5a26f1651c3cb451d3417e3bba  # v6.3.1
       with:
         version: "0.7.x"
```

Comment:
Whynot just use actions/setup-python with a pip install uv, like I did in my PR? Having hashes here increases the effort needed to maintain the pipelines.